lib/src/resultsetTable/ResultsetTable.tsx

```diff
@@ -54,15 +54,15 @@ const BothArrows = () => (
   </svg>
 );
 
-const DxcResultsetTable = ({
+const DxcResultsetTable: Overload = ({
```

Comment:
Why do u convert JSX.Element to Overload type?

Reply:
Removed pattern.